src/main/java/ua/edu/internship/interview/service/mapper/UserQuestionMapper.java

```diff
@@ -0,0 +1,24 @@
+package ua.edu.internship.interview.service.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import ua.edu.internship.interview.data.documents.UserQuestionDocument;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionCreateDto;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionDto;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionUpdateDto;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChAyNjk0OTMzNmRlNWY1MTUzEgpjaGVja3N0eWxl -->